src/main/java/de/uhd/ifi/se/decision/management/jira/quality/MetricCalculator.java

```diff
@@ -52,343 +53,365 @@
-		// TODO: Find a way to include Elements from Commit and Code as they are not
-		// linked between Issue and Decision and there is no way to link to them in the
-		// chart
-		Map<String, String> havingLinkMap = new HashMap<String, String>();
-		havingLinkMap.put(linkFrom.toString() + " has " + linkTo.toString(), data[0].trim());
-		havingLinkMap.put(linkFrom.toString() + " has no " + linkTo.toString(), data[1].trim());
-		return havingLinkMap;
+	    }
 	}
+	// TODO: Find a way to include Elements from Commit and Code as they are not
+	// linked between Issue and Decision and there is no way to link to them in the
+	// chart
+	Map<String, String> havingLinkMap = new HashMap<String, String>();
+	havingLinkMap.put(linkFrom.toString() + " has " + linkTo.toString(), data[0].trim());
+	havingLinkMap.put(linkFrom.toString() + " has no " + linkTo.toString(), data[1].trim());
+	return havingLinkMap;
+    }
 
-	public Map<String, Integer> getNumberOfRelevantComments() {
-		Map<String, Integer> numberOfRelevantSentences = new HashMap<String, Integer>();
-		int isRelevant = 0;
-		int isIrrelevant = 0;
-
-		JiraIssueTextPersistenceManager persistenceManager = KnowledgePersistenceManager.getOrCreate(projectKey)
-				.getJiraIssueTextManager();
-		for (Issue jiraIssue : jiraIssues) {
-			List<Comment> comments = ComponentAccessor.getCommentManager().getComments(jiraIssue);
-			List<KnowledgeElement> elements = persistenceManager.getElementsInJiraIssue(jiraIssue.getId());
-			for (Comment comment : comments) {
-				boolean relevant = false;
-				for (KnowledgeElement currentElement : elements) {
-					if (comment.getBody().contains(currentElement.getDescription())) {
-						relevant = true;
-						isRelevant++;
-					}
-				}
-				if (!relevant) {
-					isIrrelevant++;
-				}
-			}
-			/*
-			 * for (DecisionKnowledgeElement currentElement : elements) {
-			 * 
-			 * if (currentElement instanceof PartOfJiraIssueText && ((PartOfJiraIssueText)
-			 * currentElement).isRelevant()) { isRelevant++; } else if (currentElement
-			 * instanceof PartOfJiraIssueText && !((PartOfJiraIssueText)
-			 * currentElement).isRelevant()) { isIrrelevant++; } }
-			 */
+    public Map<String, Integer> getNumberOfRelevantComments() {
+	Map<String, Integer> numberOfRelevantSentences = new HashMap<String, Integer>();
+	int isRelevant = 0;
+	int isIrrelevant = 0;
 
+	JiraIssueTextPersistenceManager persistenceManager = KnowledgePersistenceManager.getOrCreate(projectKey)
+		.getJiraIssueTextManager();
+	for (Issue jiraIssue : jiraIssues) {
+	    List<Comment> comments = ComponentAccessor.getCommentManager().getComments(jiraIssue);
+	    List<KnowledgeElement> elements = persistenceManager.getElementsInJiraIssue(jiraIssue.getId());
+	    for (Comment comment : comments) {
+		boolean relevant = false;
+		for (KnowledgeElement currentElement : elements) {
+		    if (comment.getBody().contains(currentElement.getDescription())
+			    && currentElement.getTypeAsString() != "OTHER") {

```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use equals() to compare strings instead of '==' or '!='](https://app.codacy.com/manual/UHD/cures-condec-jira/pullRequest?prid=4909482)